boot/bootdata/hivedef.inf

```diff
@@ -1898,6 +1898,9 @@ HKCU,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon",,0x00000012
 HKCU,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","Shell",0x00020000,"%SystemRoot%\explorer.exe"
 
 HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies",,0x00000012
+HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer",,0x00000012
+HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer","NoSimpleStartMenu",0x00010003,1
```

Comment:
``SHRestricted`` looks for value firstly in HKLM. There is no Modern start menu in ReactOS, so ``NoSimpleStartMenu`` in registry should always be ``1``. I added default value to "hivedef.inf", but there values only for HKCU. In which registry-file but for HKLM I have to add default value?

Reply:
You want hivesft.inf